inc/config.php

```diff
@@ -0,0 +1,146 @@
+<?php
+/**
+ * Utility functions to retrieve and parse config files.
+ *
+ * @package altis-core
+ */
+
+namespace Altis\Core\Config;
+
+/**
+ * Retrieve the configuration for HM Platform.
+ *
+ * The configuration is defined by merging the defaults with the various files that allow to customise a particular
+ * installation.
+ *
+ * @return array Configuration data.
+ */
+function get_config() {
+	static $config;
+
+	if ( $config ) {
+		return $config;
+	}
+
+	$config = get_merged_defaults_and_customisations();
+
+	return $config;
+}
+
+/**
+ * Return the value for a particular setting from the configuration,
+ *
+ * @param string $key Settings key to retrieve the value from.
+ *
+ * @return mixed Settings value.
+ */
+function get_config_value( string $key ) {
```

Comment:
I wonder if it's worth adding a `default` arg? I think triggering an error on not found is a bit harsh to the calling code too!

Reply:
To be honest I don't think this got used anywhere in the end on the `beta` branch, going to remove it for now